kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/richmessaging/models/ALBookingDetailsModel.java

```diff
@@ -43,55 +43,55 @@ public String getTitle() {
         }
 
         public void setTitle(String title) {
-            Title = title;
+            Title = title.trim();
         }
 
         public String getAge() {
             return Age;
         }
 
         public void setAge(String age) {
-            Age = age;
+            Age = age.trim();
         }
 
         public String getFirstName() {
             return FirstName;
         }
 
         public void setFirstName(String firstName) {
-            FirstName = firstName;
+            FirstName = firstName.trim();
         }
 
         public String getMiddleName() {
             return MiddleName;
         }
 
         public void setMiddleName(String middleName) {
-            MiddleName = middleName;
+            MiddleName = middleName.trim();
         }
 
         public String getLastName() {
             return LastName;
         }
 
         public void setLastName(String lastName) {
-            LastName = lastName;
+            LastName = lastName.trim();
         }
 
         public String getEmailId() {
             return EmailId;
         }
 
         public void setEmailId(String emailId) {
-            EmailId = emailId;
+            EmailId = emailId.trim();
         }
 
         public String getPhoneNo() {
             return PhoneNo;
         }
 
         public void setPhoneNo(String phoneNo) {
-            PhoneNo = phoneNo;
+            PhoneNo = phoneNo.trim();
```

Comment:
These values will always be not null? Otherwise it will throw nullpointerexception 
Also, pls check why the properties are starting with Capital. Is it existing code?

Reply:
These aren't the generic rich messages that are in the kommuniate docs. @reytum will know if somone uses there types of messages.